src/5e-SRD-Magic-Items.json

```diff
@@ -21,12 +25,86 @@
       "name": "Ammunition",
       "url": "/api/equipment-categories/ammunition"
     },
+    "rarity": {
+      "name": "Varies"
+    },
+    "variants": [
+      {
+        "index": "ammunition-1",
+        "name": "Ammunition, +1",
+        "url": "/api/magic-items/ammunition-1"
+      },
+      {
+        "index": "ammunition-2",
+        "name": "Ammunition, +2",
+        "url": "/api/magic-items/ammunition-2"
+      },
+      {
+        "index": "ammunition-3",
+        "name": "Ammunition, +3",
+        "url": "/api/magic-items/ammunition-3"
+      }
+    ],
     "desc": [
       "Weapon (any ammunition), uncommon (+1), rare (+2), or very rare (+3)",
       "You have a bonus to attack and damage rolls made with this piece of magic ammunition. The bonus is determined by the rarity of the ammunition. Once it hits a target, the ammunition is no longer magical."
     ],
     "url": "/api/magic-items/ammunition"
   },
+  {
+    "index": "ammunition-1",
```

Comment:
What are your thoughts on putting the variants into their own json file. 

Could be something like `5e-SRD-Magic-Item-Varients.json` ? That would allow this table to easily still be used to pull up items as they are in the DMG, while giving people the flexibility to pull the individual variants up. 

As it stands the only way to filter out the variants is to pull the whole list then the consumer would have to loop over all the items and find out what to remove on their end.

Reply:
It's certainly doable to keep everything in this file, I could just see it being a bit more complicated for people who are using it. I don't think it's a deal breaker either way though.

Reply:
So, on the one hand, I can see the value in splitting that out like that. This also allows the api to do: `/api/magic-items/ammunition/variants` to give you only the variants.

On the other hand, it becomes a lot hard to get all the magic items of a certain rarity.

Reply:
Does it become that much harder? You could just query both tables and join combine the results.

Reply:
It doesn't. How about this? What if you added a `variant: Boolean` field into all the entries. This would allow you keep it together but also allow you to easily query for things that aren't variants?

Reply:
Flags work when they make sense.

Reply:
added a variant flag to each item